brave/brave-core
# Add Wallet backup banner and button

Resolves <https://github.com/brave/brave-browser/issues/16077>

## Submitter Checklist:

1) Create a Backup Wallet Banner warning that will display if the user decides to skip the backup process during onboarding.
2) Add a backup button in the wallet more popup menu.
3) Separated `Backup Wallet` from onboarding to make it reusable later. 


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that storybook shows the new component.
3. Check functionality of the new component.

https://user-images.githubusercontent.com/40611140/120049427-27b55080-bfd7-11eb-8da2-f35faade962d.mov
